Classes/Neos/Neos/Ui/Domain/Model/Feedback/Operations/Redirect.php

```diff
@@ -0,0 +1,101 @@
+<?php
+namespace Neos\Neos\Ui\Domain\Model\Feedback\Operations;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Neos\Service\LinkingService;
+use Neos\Neos\Ui\Fusion\Helper\NodeInfoHelper;
+use Neos\Neos\Ui\Domain\Model\FeedbackInterface;
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+use Neos\Flow\Mvc\Controller\ControllerContext;
+
+class Redirect implements FeedbackInterface
+{
+    /**
+     * @var NodeInterface
+     */
+    protected $node;
+
+    /**
+     * @Flow\Inject
+     * @var NodeInfoHelper
+     */
+    protected $nodeInfoHelper;
+
+    /**
+     * @Flow\Inject
+     * @var LinkingService
+     */
+    protected $linkingService;
+
+    /**
+     * Set the node
+     *
+     * @param NodeInterface $node
+     * @return void
+     */
+    public function setNode(NodeInterface $node)
+    {
+        $this->node = $node;
+    }
+
+    /**
+     * Get the node
+     *
+     * @return NodeInterface
+     */
+    public function getNode()
+    {
+        return $this->node;
+    }
+
+    /**
+     * Get the type identifier
+     *
+     * @return string
+     */
+    public function getType()
+    {
+        return 'Neos.Neos.Ui:Redirect';
+    }
+
+    /**
+     * Get the description
+     *
+     * @return string
+     */
+    public function getDescription()
+    {
+        return sprintf('Redirect to node "%s".', $this->getNode()->getContextPath());
```

Comment:
input type and return type declaration are missing at many placed, would be nice to have them